<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AMIT" failures="2" tests="2" name="FirefoxTest" time="14.522" errors="0" timestamp="2022-07-21T21:03:03 IST">
  <testcase classname="com.mystore.testscripts.IndexPageTest" name="VerifyTitle" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [My Store] but found [508 Resource Limit Is Reached]">
      <![CDATA[java.lang.AssertionError: expected [My Store] but found [508 Resource Limit Is Reached]
at com.mystore.testscripts.IndexPageTest.VerifyTitle(IndexPageTest.java:44)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- VerifyTitle -->
  <testcase classname="com.mystore.testscripts.IndexPageTest" name="VerifymyStoreLogo" time="10.019">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.mystore.testscripts.IndexPageTest.VerifymyStoreLogo(IndexPageTest.java:33)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- VerifymyStoreLogo -->
</testsuite> <!-- FirefoxTest -->
